heatpump.yaml

```diff
@@ -1712,152 +1730,28 @@ binary_sensor:
-    icon: mdi:eye
-    register_type: holding
-    address: 0xd2
-    bitmask: 0x4
-  # Bit: 3, default: true, TODO: verify default
-  - platform: modbus_controller
-    modbus_controller_id: "${devicename}"
-    name: "Parameter Setting 1 Supports Room Thermostat"
-    id: "${devicename}_parameter_setting_1_supports_room_thermostat"
-    icon: mdi:eye
-    register_type: holding
-    address: 0xd2
-    bitmask: 0x8
-  # Bit: 4, default: false, TODO: verify default
-  - platform: modbus_controller
-    modbus_controller_id: "${devicename}"
-    name: "Parameter Setting 1 Supports Room Temperature Sensor Ta"
-    id: "${devicename}_parameter_setting_1_supports_room_temperature_sensor_ta"
-    icon: mdi:eye
-    register_type: holding
-    address: 0xd2
-    bitmask: 0x10
-  # Bit: 5, default: false, TODO: verify default
-  # Midea: PUMPI silent mode, 1; valid, 0: invalid
-  - platform: modbus_controller
-    modbus_controller_id: "${devicename}"
-    name: "Parameter Setting 1 Supports T1 Sensor"
-    id: "${devicename}_parameter_setting_1_supports_t1_sensor"
-    icon: mdi:eye
-    register_type: holding
-    address: 0xd2
-    bitmask: 0x20
-  # Bit: 6, default: false, TODO: verify default
-  - platform: modbus_controller
-    modbus_controller_id: "${devicename}"
-    name: "Parameter Setting 1 T1S Heating High Low Temperature Settings"
-    id: "${devicename}_parameter_setting_1_t1s_heating_high_low_temperature_settings"
-    icon: mdi:eye
-    register_type: holding
-    address: 0xd2
-    bitmask: 0x40
-  # Bit: 7, default: true, TODO: verify default
-  - platform: modbus_controller
-    modbus_controller_id: "${devicename}"
-    name: "Parameter Setting 1 Enable Heating"
-    id: "${devicename}_parameter_setting_1_enable_heating"
-    icon: mdi:eye
-    register_type: holding
-    address: 0xd2
-    bitmask: 0x80
-  # Bit: 8, default: false, TODO: verify default
-  - platform: modbus_controller
-    modbus_controller_id: "${devicename}"
-    name: "Parameter Setting 1 T1s Cooling High Low Temperature Settings"
-    id: "${devicename}_parameter_setting_1_t1s_cooling_high_low_temperature_settings"
-    icon: mdi:eye
-    register_type: holding
-    address: 0xd2
-    bitmask: 0x100
-  # Bit: 9, default: true, TODO: verify default
-  - platform: modbus_controller
-    modbus_controller_id: "${devicename}"
-    name: "Parameter Setting 1 Enable Cooling"
-    id: "${devicename}_parameter_setting_1_enable_cooling"
-    icon: mdi:eye
-    register_type: holding
-    address: 0xd2
-    bitmask: 0x200
-  # Bit: 10, default: false, TODO: verify default
-  - platform: modbus_controller
-    modbus_controller_id: "${devicename}"
-    name: "Parameter Setting 1 DHW Pump Supports Pipe Disinfect"
-    id: "${devicename}_parameter_setting_1_dhw_pump_supports_pipe_disinfect"
-    icon: mdi:eye
-    register_type: holding
-    address: 0xd2
-    bitmask: 0x400
+  # Register: 210, default: true, TODO: verify default  -> moved to switch
```

Comment:
I added at some point the `is present in this config` remarks, if a few parts of a register or registers from a specific range where handled differently. Since now almost the complete register 210 is moved, this line and the line for bit [0-10] and bit [12-15] can be removed.

Then at the place where register 210 is implemented, a specific comment line for bit 11 will explain that only bit 11 is a binary_sensor

Reply:
Please also remove line 1734: `# Register: 210, default: true, TODO: verify default -> moved to switch`